pgmpy/tests/test_readwrite/test_ProbModelXML.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+import unittest
+from io import StringIO
+import networkx as nx
+from pgmpy.readwrite import ProbModelXMLWriter, ProbModelXMLReader, parse_probmodelxml, generate_probmodelxml
+try:
+    from lxml import etree
+except ImportError:
+    try:
+        # Python 2.5
```

Comment:
In test_PomdpX also it is mentioned liked this. Therefore i have used it.
https://github.com/pgmpy/pgmpy/blob/dev/pgmpy/tests/test_readwrite/test_PomdpX.py#L14
Should i change it in that test file also ?